### ISSUE TITLE
loss.backward()

### ISSUE THREAD

**Issue by liuchuan19971008** [chengxuxin/extreme-parkour#41]
Hello!
I have a problem.The forward frequency of depth_encoder and depth_actor are different in learn_vision.The depth_encoder is 10 Hz, but depth_actor is 50 Hz. Why there is no problem, when depth_loss.backward() if I use "depth_encoder_loss + depth_actor_loss".

**Comment by zma69650** [comment]
@liuchuan19971008   请问问题解决了吗
